docs/configuring-metabase/localization.md

```diff
@@ -6,6 +6,54 @@ redirect_from:
 
 # Languages and localization
 
+The **Localization** settings allow you to set global defaults for your Metabase instance. You can find **Localization** under **Admin settings** > **Settings**.
+
+## Instance language
+
+The instance language is the default language across the Metabase UI, system [emails](./email.md), [dashboard subscriptions](../dashboards/subscriptions.md), and [alerts](../questions/sharing/alerts.md). People can pick a different language from their own [account settings](../people-and-groups/account-settings.md).
+
+## Report timezone
+
+Use **report timezone** to set a default display time zone for dates and times in Metabase. This is a display setting only, so changing the **report timezone** won't affect the time zone of any data in your database.
```

Comment:
```suggestion
Use **report timezone** to set a default display time zone for dates and times in Metabase. The report timezone setting is a display setting only, so changing the report timezone won't affect the time zone of any data in your database. 
```

Yes, the revision is a tad tedious, prose-wise, but the edit imposes a somewhat lighter cognitive load than forcing people to infer what we mean by "this". Plus repeating the term reinforces the terminology. Side note: we only need to bold a key term the first time we use it.